composer.json

```diff
@@ -1,21 +1,54 @@
 {
-  "name": "ibexa/phpstan",
-  "license": "proprietary",
-  "type": "ibexa-bundle",
-  "keywords": [
-    "ibexa-dxp"
-  ],
-  "require": {
-    "php": "^7.4 || ^8.0"
-  },
-  "config": {
-    "sort-packages": true
-  },
-  "extra": {
-    "phpstan": {
-      "includes": [
-        "extension.neon"
-      ]
+    "name": "ibexa/phpstan",
+    "license": "proprietary",
+    "type": "ibexa-bundle",
+    "keywords": [
+        "ibexa-dxp"
+    ],
+    "require": {
+        "php": "^7.4 || ^8.0",
+        "ibexa/core": "4.6.x-dev",
```

Comment:
So that means we have to have separate branches: `4.6` and `main`, right?

Reply:
You are right, adjusted.